src/App.js

```diff
@@ -1,13 +1,95 @@
-import React from 'react';
+import React from "react";
+import TodoList from "./components/TodoComponents/TodoList";
+import TodoForm from "./components/TodoComponents/TodoForm";
+import "./css/main.css";
+
+// I put this here so I wouldn't have to populate the task list to test code:
+const taskList = [
+  {
+    task: "organize garage",
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: "bake cookies",
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      taskList: taskList,
+      task: ""
+    };
+  }
```

Comment:
it's my bad for not pointing this out already, but since there are no `props` being passed down into this App component (i.e., it's not nested) you actually don't need to use constructor or super here, and can just write `state` as a variable instead of `this.state` inside the constructor. it still works, but is not needed unless props are being passed down. still useful to be familiar with how this constructor method looks in case you have nested class components